debezium-parent/pom.xml

```diff
@@ -25,7 +25,7 @@
         <skipITs>false</skipITs>
 
         <!-- Do not skip formatting source code by default -->
-        <format.skip>false</format.skip>
+        <format.skip>true</format.skip>
```

Comment:
Added since, on runing maven commands, by default a lot of files got changed and hence relevant changes were lost.